content/cloud-docs/agents/hooks.mdx

```diff
@@ -0,0 +1,137 @@
+---
+page_title: Hooks - Terraform Cloud Agents - Terraform Cloud and Terraform Enterprise
```

Comment:
Description helps with SEO! And it can show up as the snippet that google shows folks in search results. 

```suggestion
page_title: Hooks - Terraform Cloud Agents - Terraform Cloud and Terraform Enterprise
description: >-
  Hooks are custom programs that run at specific points during Terraform operations. 
```